Source/Fuse.Maps/iOS/MapViewDelegate.m

```diff
@@ -214,6 +224,48 @@ -(void) clearMarkers
 			[_mapView removeAnnotation:a];
 		}
 		[_annotations removeAllObjects];
+
+		for(id key in _overlays)
+		{
```

Comment:
This should be `for (id key in _overlays) {` for consistency with the surrounding code...